lib/spack/spack/build_systems/oneapi.py

```diff
@@ -9,11 +9,10 @@
 import shutil
 from os.path import basename, dirname, isdir
 
-from llnl.util.filesystem import find_headers, find_libraries, join_path
+from llnl.util.filesystem import find_headers, find_libraries, join_path, mkdirp
```

Comment:
where is this used?

Reply:
For sure it should not be imported from `spack.package`. I wish Python was as strict as Go for circular imports